### ISSUE TITLE
Cant get edited content

### ISSUE THREAD

**Issue by khilkevichigor** [FlowingCode/MarkdownEditor#9]
### Describe the bug

I have this:
`<vaadin.version>24.5.0</vaadin.version>`

and this from https://vaadin.com/directory/component/markdown-editor-add-on
```
<dependency>
   <groupId>com.flowingcode.vaadin.addons</groupId>
   <artifactId>markdown-editor-addon</artifactId>
   <version>1.0.0</version>
</dependency>
```

And I use this:
```
MarkdownEditor mde = new MarkdownEditor();
mde.setSizeFull();
mde.setHeight("300px");
mde.setMaxLength(500);
mde.setDataColorMode(BaseMarkdownComponent.DataColorMode.LIGTH);
mde.setContent(post.getPostText);//"aaa"
formLayout.add(mde);
```

I then change the text on the UI in mde from "aaa" to "bbb" and use that afterwards:

```
dialog.addDialogCloseActionListener(closeEvent -> {
      post.setPostText(mde.getContent()); //here for some reason i am getting "aaa" instead of "bbb"!
      postService.save(post);
});
```

### Expected behavior

i expect to get "bbb"

### Add-on Version

1.0.0

### Vaadin Version

24.5.0

**Comment by mlopezFC** [comment]
This is a duplicate of #4 . Already fixed in #6 . We're going to do the release soon and keep you posted.
Regards.

**Comment by mlopezFC** [comment]
We released [a new version](https://vaadin.com/directory/component/markdown-editor-add-on/1.1.0) containing this fix. Closing this one, but if you have any trouble just reopen it or create a new issue.
